### PR TITLE
Better TCPA Rebuttals and Such

### PR DESCRIPTION
**Pull request by MTS11648**
Read above.

### CODE REVIEW

**Comment by MTS11648** [comment]
Do NOT merge until you've merged #183.

File: AutoBotCSharp/Agent.cs [hunk 1]
```diff
@@ -290,14 +290,15 @@ public static void onPlaybackStopped(object sender, StoppedEventArgs e)
         }
 
         public static int cherylRepeats = 0; // Times AutoFormQuestion has been asked by Cheryl
+        public static int cherylTCPARebuttals = 0; //Times Cheryl has countered a "No" to the TCPA.
```
Comment (SpadesNeil) [inline AutoBotCSharp/Agent.cs]: This variable does not appear to be reset at the beginning of calls.

Comment (SpadesNeil) [inline AutoBotCSharp/Agent.cs]: Or not. Apparently that isn't how it works.

**Comment by MTS11648** [comment]
Merging because this has been running for a very long time as far as I know and I haven't seen any issues with it yet.

**Pull request merged**